data-prepper-pipeline-parser/src/main/java/org/opensearch/dataprepper/validation/PluginErrorsConsolidator.java

```diff
@@ -0,0 +1,18 @@
+package org.opensearch.dataprepper.validation;
+
+import javax.inject.Named;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+@Named
+public class PluginErrorsConsolidator {
```

Comment:
I think rather than make this a consolidator that creates a string, a more flexible approach would be to make a `PluginErrorsHandler`. Let it decide how to handle multiple errors. Maybe it handles it by creating a structured object. Maybe it handles it by logging. Maybe by stdout.

```
public interface PluginErrorsHandler {
  public void handleErrors(final Collection<PluginError> pluginErrors);
}
```

And then we can have a default which logs:

```
@Named
public class LoggingPluginErrorsHandler implements PluginErrorsHandler {
  public void handleErrors(final Collection<PluginError> pluginErrors) {
    // build string
    // log to Slf4j
  }
}
```